.env.sample

```
@@ -30,4 +30,5 @@ S3_UPLOAD_REGION=""
 # Misc
 PROCESSING_FEE="26"
 CONFIRMATION_EMAIL_FROM=""
-INVOICE_LINK_TTL_DAYS="7"
\ No newline at end of file
+INVOICE_LINK_TTL_DAYS="7"
+APPLICATION_DOCUMENT_LINK_TTL_DAYS="7"
```

Oustan mentioned the TTL for documents should be 1 hour so the env var should probably be changed to 
```suggestion
APPLICATION_DOCUMENT_LINK_TTL_HOURS="1"
```

Then to convert into seconds for the signed url helper function:
```js
const durationSeconds = parseInt(process.env.APPLICATION_DOCUMENT_LINK_TTL_HOURS) * 60 * 60;
signedUrl = getSignedUrlForS3(documentsUrl, durationSeconds);
```